src/uu/rev/src/rev.rs

```diff
@@ -0,0 +1,85 @@
+// This file is part of the uutils util-linux package.
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+use clap::{crate_version, Command};
+use clap::{Arg, ArgAction};
+use std::env;
+use std::io::{BufRead, BufReader, Read, Write};
+use uucore::{error::UResult, format_usage, help_about, help_usage};
+
+const ABOUT: &str = help_about!("rev.md");
+const USAGE: &str = help_usage!("rev.md");
+
+#[uucore::main]
+pub fn uumain(args: impl uucore::Args) -> UResult<()> {
+    let matches: clap::ArgMatches = uu_app().try_get_matches_from(args)?;
+    let files = matches.get_many::<String>("file");
+    let mut has_error = false;
+
+    match files {
+        Some(files) => {
+            for path in files {
+                let file = match std::fs::File::open(path) {
+                    Ok(val) => val,
+                    Err(err) => {
+                        has_error = true;
+                        eprintln!("rev: cannot open {}: {}", path, err);
```

Comment:
You can simplify the error handling by using either `show!` or `show_error!` from `uucore`.

Reply:
The `eprintln!`s are replaced by `show_error!` in latest commit, thanks.

For `has_error`, I observed `util-linux`'s `rev`'s behavior is:

 * If any error occurred, it immediately prints the error, but not terminating the process, instead, it continues to process files passed from command-line arguments;
 * If any error occurred, the final exit code will be `1`.
 * If all files are `rev`-ed successfully, the final exit code will be `0`.

That's why `has_error` was set.